README.md

```diff
@@ -68,8 +68,8 @@ for episode in range(N_EPISODES):
 ### Environment and Task configurations
 
 The environment is typically defined by a JSON configuration file located on the `gym_ai2thor/config_files` 
-folder. You can find an example `config_example.json` to see how to customize it. Here there is one
-as well:
+folder. You can find a full example at `config_example.json` to see how to customize it. Here there is 
```

Comment:
Generally from the readme and not this particular line: should we explain something of continuous rotation in here? In the official website and repo it is not documented. I just found it looking at the issues in the repo

Reply:
Also reminding myself to change the name to Unified Physical

Reply:
add env.close to all test cases
create test case which will just create each task just to make sure they still can be created

Reply:
Added that github issue to the docstring in 
task_on_ground_continuous_random_walk.py